backend/MeetApp/src/main/java/com/meetapp/meetapp/model/Event.java

```diff
@@ -30,12 +30,10 @@ public class Event extends Post {
 
     @NotNull
     @Basic
-    @Future
     @Column(nullable = false)
     private Instant startDate;
 
     @NotNull
-    @Future
```

Comment:
Since you deleted those @Instant's, shouldn't we now add some date checking logic to the post methods in Service classes? :nerd_face: